great_expectations/expectations/expectation.py

```diff
@@ -337,6 +337,7 @@ class Config:
 
     id: Union[str, None] = None
     meta: Union[dict, None] = None
+    notes: str = ""
```

Comment:
`str = ""` or `str | None = None`?

Reply:
Leaning towards the former - let's avoid nullable types if possible

Reply:
My preference is for the latter. I like the type to tell me about the intended absence of something.